db/seeds.rb

```diff
@@ -5,9 +5,18 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
-FoodTruck.create(
-  name: "Chicken & Rice Guys",
-  description: "Food truck that specializes in chicken and lamb shawarma.",
-  avg_rating: 4,
-  location: "Harvard Square"
-)
+
+foodtrucks = [
+    ["Chicken & Rice Guys", "Food truck that specializes in
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.